client/src/style.css

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
Can drive these global configurations from styled components, just so you can keep everything the same and using themes etc